contracts/BSCValidatorSet.sol

```diff
@@ -56,6 +58,14 @@ contract BSCValidatorSet is IBSCValidatorSet, System, IParamSubscriber, IApplica
   uint256 public burnRatio;
   bool public burnRatioInitialized;
 
+  // BEP-99 Temporary Maintenance
```

Comment:
`BEP-99` ==> `BEP-127`, let's search all BEP-99, and replace them

Reply:
fixed